### ISSUE TITLE
hardsuit needs charging/suit charger 

### ISSUE THREAD

**Issue by Emisse** [space-wizards/space-station-14#23214]
not having it charged should slow you/disable features or something, suit rechargers should be accessible and maybe we do the battery thing again and see how it goes.
rn its no different from softsuits/voidsuits besides toggleable helmet.
could also port modsuits as a sidegrade/engineering thing not a REPLACEMENT though.

**Comment by MACMAN2003** [comment]
i feel like the addition of another slowly depleting thing to juggle would make it painful for jobs that spend a lot of time in hardsuits like engies, especially if they have to respond immediately to something and they're far away from a recharger.

in other words, the thirst mechanic is already punishing enough for engies :(

**Comment by K-Dynamic** [comment]
Hardsuit charge mechanic would only make sense with abilities. Currently they only have spacing protection, damage resistances and weak lights.

And if low charge slows you as proposed, then regular EVA suits will be straight-up better; -25% speed for EVA room suits without needing charge, -15% to -25% speed on most hardsuits _before accounting for low charge speed penalties_. EMPs would also become punishing as hell for hardsuit users.

No idea how this interact with bloodred and other antag hardsuits, maybe they won't kill station power so they can charge as well

**Comment by keronshb** [comment]
I do not agree with this. What is the scenerio where hardsuits would actually need recharging? They're not even that much of an issue in SS13 where they're even less restricted.

**Comment by keronshb** [comment]
> I don't follow. You'd want to keep the hardsuit charged to avoid the penalties that come when its batteries run out... or are you asking how frequently / where the hardsuits would be charged?

Unless there's some sort of non-antag hardsuit that gives an exceptional advantage, what does even giving charge for them do in the first place? All it does is make Engineer/Atmos' job that much harder.

Crew Hardsuits are effectively spacesuits that have a flashlight on the helmet. Maybe sec/captain/HOS might have armor on them and the RD one might be bomb resistant, but overall they're not that insanely broken that they need slowdown + held slowdown + charge.
